machine_management/control_plane_machine_management/cpmso-about.adoc

```diff
@@ -22,8 +22,23 @@ include::modules/cpmso-overview.adoc[leveloffset=+1]
 [id="cpmso-limitations_{context}"]
 == Limitations
 
-In {product-title} {product-version}, the Control Plane Machine Set Operator has the following limitations:
+The Control Plane Machine Set Operator has the following limitations:
+
+* The Operator requires the Machine API to be operational and is therefore not supported on clusters with manually provisioned machines.
+
+* Only Amazon Web Services (AWS), Microsoft Azure, and VMware vSphere clusters are supported.
 
 * Only clusters with three control plane machines are supported.
 
 * Horizontal scaling of the control plane is not supported.
+
+* Deploying Azure control plane machines on Ephemeral OS disks increases risk for data loss and is not supported.
+
+* Deploying control plane machines as AWS Spot Instances or Azure Spot VMs is not supported.
++
+[IMPORTANT]
+====
+Attempting to deploy control plane machines as AWS Spot Instances or Azure Spot VMs can cause catastrophic failure.
```

Comment:
Thanks, folks :) Thoughts on this?
> Attempting to deploy control plane machines as AWS Spot Instances or Azure Spot VMs can cause the cluster to lose etcd quorum. A cluster that loses all control plane machines simultaneously is unrecoverable.

Reply:
Looks good, though one nit, if I were writing this I would have put `may cause` rather than `can cause`, they have a different connotation to me but that could be a British vs American english thing